contexts/ConfigContext.tsx

```diff
@@ -38,14 +65,19 @@ export async function fetchConfigFromAPI(): Promise<RuntimeConfig> {
       )
       return defaultConfig
     })
+
+  return {
+    ...defaultConfig,
```

Comment:
I don't think we should merge `defaultConfig` with the API configuration because `defaultConfig` will be full of empty values when running as a static bundle.